gnomad/utils/sparse_mt.py

```diff
@@ -319,11 +319,16 @@ def get_as_info_expr(
     if "AS_SB" in agg_expr:
         agg_expr["AS_SB_TABLE"] = agg_expr.pop("AS_SB")
 
+    if 'alt_alleles_range_array' not in mt.row or mt['alt_alleles_range_array'].dtype != hl.dtype('array<int32>'):
```

Comment:
I think I'd like it better to have this as an argument with a default value. Or alternatively mention what this array is in the doc. Happy to modify but depends what the team thinks in terms of interface.